src/mesh.c

```diff
@@ -10,5 +10,20 @@ PetscErrorCode CreateMesh(MPI_Comm comm, DM *dm, PetscBool simplex, PetscInt dim
 
     ierr = DMViewFromOptions(*dm, NULL, "-dm_view");CHKERRQ(ierr);
 
+    // distribute the mesh
```

Comment:
This is fine, and matches prior examples. However, I am moving to an interface where distribution is handled automatically in DMSetFromOptions() using the -dm_distribute option. This means less code in the application to maintain. We can always do this later, but I thought I would mention it.

Reply:
Thanks!  I removed the manual distribute and added the option to the test.